paddlepaddle/paddlepaddle-engine/src/main/java/ai/djl/paddlepaddle/engine/PpSymbolBlock.java

```diff
@@ -18,23 +18,26 @@
 import ai.djl.ndarray.types.Shape;
 import ai.djl.nn.AbstractBlock;
 import ai.djl.nn.SymbolBlock;
-import ai.djl.paddlepaddle.jna.JnaUtils;
+import ai.djl.paddlepaddle.jni.JniUtils;
 import ai.djl.training.ParameterStore;
 import ai.djl.util.PairList;
+import java.util.Arrays;
 
 /** {@code PpSymbolBlock} is the PaddlePaddle implementation of {@link SymbolBlock}. */
-public class PpSymbolBlock extends AbstractBlock implements SymbolBlock {
+public class PpSymbolBlock extends AbstractBlock implements SymbolBlock, AutoCloseable {
```

Comment:
We don't need make this AutoCloseable.

The PaddlePredictor will be closed in PpPredictor and PpModel.